cargo-workspaces/src/init.rs

```diff
@@ -0,0 +1,75 @@
+use std::collections::HashSet;
+use std::fs;
+
+use cargo_metadata::MetadataCommand;
+use clap::Clap;
+use glob::glob;
+
+use crate::utils::info;
+use crate::utils::{Error, Result};
+use std::path::PathBuf;
```

Comment:
Can you please combine all the use statements without any empty newlines between them? Example:

```rust
use std::{collections::HashSet, fs};
```